doc review: traffic split qs (#47)
opendatahub-io/odh-model-controller: docs/quickstart-traffic-splitting.md

# Traffic splitting quick start

This `odh-model-controller` quick start demonstrates how to use the ModelMesh framework with traffic splitting. 

## Prerequisites

- Install ModelMesh as described in the [ModelMesh serving quickstart's installation section](https://github.com/opendatahub-io/modelmesh-serving/blob/main/docs/quickstart.md#1-install-modelmesh-serving). The `odh-model-controller` is a companion controller for [modelmesh-serving](https://github.com/opendatahub-io/modelmesh-serving).

- Install Service Mesh depending on your OpenShift or Kubernetes system.

  - On OpenShift, use the [deploy.ossm.sh](../scripts/deploy.ossm.sh) script to quickly install OpenShift Service Mesh (OSSM). On Red Hat OpenShift for AWS, you must uncomment a few lines in the `deploy.ossm.sh` script before you run it.

>   - On another type of OpenShift (or Kubernetes), use [Istio](https://istio.io/) to install OSSM. You must also install the required `istiod` and `istio-ingressgateway` components. For example, if you [install with `istioctl`](https://istio.io/latest/docs/setup/install/istioctl/), use the `istioctl install` command which also installs `istiod` and `istio-ingressgateway`.

Reviewer: Istio is the upstream project of OSSM. So, we cannot say that Istio is used to install OSSM. We just install Istio and that's it. Istio works in (almost?) any Kubernetes system (including OpenShift), while OSSM works only in OpenShift.  The `istiod` and `istio-ingressgateway` components are part of Istio and can be installed selectively.

Additionally, I just remembered that this repository is OpenShift specific and not meant for any other Kubernetes system.

I tried to fix this second bullet to denote all this, but I think my suggestion needs you refinement:

```suggestion
  - Alternatively, use [Istio](https://istio.io/). You must install the required `istiod` and `istio-ingressgateway` components. For example, if you [install with `istioctl`](https://istio.io/latest/docs/setup/install/istioctl/), using the `istioctl install` command includes the required `istiod` and `istio-ingressgateway` components.
```

        - '*'
        port:
          name: http

>           number: 8

Reviewer: This seems to be a change by mistake. Please, fix.

          protocol: HTTP
    EOF
    ```
    
1. Install the ODH-Model-Controller:

    ```shell
    # Clone the odh-model-controller repository
    git clone https://github.com/opendatahub-io/odh-model-controller.git

    # Enter into the cloned code path
    cd odh-model-controller

    # Deploy odh-model-controller
    make deploy -e IMG=quay.io/edgarhz/odh-model-controller:service-mesh-integration
    ```

> 1. Enroll both the `opendatahub` and this `modelmesh-serving` namespaces in the mesh:

Reviewer: This is only for OSSM. For Istio, this step is not done. So, I think we should keep the `If you are using OSSM,` text (or some more elegant writing)
